source/guides/mattermost-ai-copilot-admin-guide.rst

```diff
@@ -0,0 +1,74 @@
+Admin Guide: Mattermost AI Copilot Plugin
```

Comment:
> Mattermost AI Copilot Plugin

Do we need to identify this as a "plugin" in the title. My raise would be that we call this and the end user guide just "... AI Copilot". Then for the Admin guide we mention how it's a plugin that needs to be installed (if on <v9.7). If on >v9.7 they don't need to install it since it's prepackaged. 

For the end user guide it's not important that we mention it's a plugin at all, it's part of their Mattermost experience if your admin has enabled it.